### PR TITLE
[feature] sms: delete 'undo' function and add responsive design

### PR DESCRIPTION
**Pull request by fcampo**
- Getting rid of Undo functionality as is not on new UX wireframes
- Change CSS from px to rem units for responsive design (10px base for SG2 and 7.3px base for Akami). Still need to sync with rest of the apps

### CODE REVIEW

File: apps/sms/index.html [hunk 1]
```diff
@@ -63,14 +61,14 @@ <h2 id="view-name">
       </div>
       <div id="msg-conversations-list" class="list">
       </div>
-      <div id="msg-undo-toolbar" class="toolbar">
+      <!-- <div id="msg-undo-toolbar" class="toolbar">
```
Comment (arcturus) [inline apps/sms/index.html]: We should remove this code instead of commenting it

File: apps/sms/js/sms.js [hunk 2]
```diff
@@ -320,37 +276,25 @@ var ConversationListView = {
...
-    this.undoToolbar.classList.remove('show');
     this.deleteMessages(this.delNumList);
     this.delNumList = [];
   },
-
-  undoMessageDelete: function cl_undoMessageDelete() {
-    DelayDeleteManager.unregistDelayDelete();
+  
+  executeAllMessagesDelete: function cl_executeAllMessagesDelete() {
+    console.log("*********** NUMLIST ANTES= "+this.delNumList.length);
```
Comment (arcturus) [inline apps/sms/js/sms.js:286]: Logs should be avoided

File: apps/sms/js/sms.js [hunk 3]
```diff
@@ -320,37 +276,25 @@ var ConversationListView = {
...
+    this.delNumList = // coger todos los mensajes existentes
+    console.log("*********** NUMLIST DESPUES= "+this.delNumList.length);
```
Comment (arcturus) [inline apps/sms/js/sms.js:288]: Logs again :P

File: apps/sms/style/sms.css [hunk 4]
```diff
@@ -368,43 +369,43 @@ body.msg-edit-mode #msg-conversations-list > a {
...
 
 body.msg-edit-mode #msg-conversations-list .time {
-  right: 106px;
+  right: 10.6rem;
 }
 
-#msg-undo-toolbar {
-  height: 80px;
-  bottom: 80px;
+/*#msg-undo-toolbar {
```
Comment (arcturus) [inline apps/sms/style/sms.css]: Remove this code if not used

File: apps/sms/style/sms.css [hunk 5]
```diff
@@ -368,43 +369,43 @@ body.msg-edit-mode #msg-conversations-list > a {
...
+  height: 8rem;
+  bottom: 8rem;
   background: url("images/app-texture.png") repeat scroll 50% 50%, -moz-linear-gradient(center top , #000, #444) repeat scroll 0% 0% transparent;
   visibility: hidden;
-}
+}*/
 
-#msg-undo-toolbar.show {
+/*#msg-undo-toolbar.show {
```
Comment (arcturus) [inline apps/sms/style/sms.css]: Remove this code if not used

File: apps/sms/style/sms.css [hunk 6]
```diff
@@ -368,43 +369,43 @@ body.msg-edit-mode #msg-conversations-list > a {
...
   visibility: visible;
-}
+}*/
 
-#msg-undo-title-container {
+/*#msg-undo-title-container {
```
Comment (arcturus) [inline apps/sms/style/sms.css]: Remove this code if not used

File: apps/sms/style/sms.css [hunk 7]
```diff
@@ -568,14 +608,14 @@ body.conversation-new-msg #view-num {
   }
 }
 
-#view-list .photo {
+/*#view-list .photo {
```
Comment (arcturus) [inline apps/sms/style/sms.css:592]: Remove this code if not used

File: gaia [hunk 8]
```diff
@@ -1 +1 @@
-Subproject commit defc5503c4acb03f226905370ba83a062e4adb17
+Subproject commit 356f65380cb015885a588a815b6791a8e3c9ca69
```
Comment (ferjm) [inline gaia:1]: Do not include gaia changes on your pull request, please

File: apps/sms/js/sms.js (hunk 3 above)
Comment (arcturus) [inline apps/sms/js/sms.js:288]: Fernando C is working on this and requires the logs. Will  be removed in next commits.

File: apps/sms/js/sms.js (hunk 2 above)
Comment (arcturus) [inline apps/sms/js/sms.js:286]: Fernando C is working on this and requires the logs. Will  be removed in next commits.

File: gaia (hunk 8 above)
Comment (fcampo) [inline gaia:1]: Fixed :D (I hope)

File: apps/sms/style/sms.css (hunk 7 above)
Comment (fcampo) [inline apps/sms/style/sms.css:592]: Still working on photo for conversations (previously on single message)

File: gaia (hunk 8 above)
Comment (ferjm) [inline gaia:1]: nope https://github.com/telefonicaid/OpenWebDevice/pull/15/files

**Pull request closed without merging**